Formula/qscintilla2.rb

```diff
@@ -12,14 +12,17 @@ class Qscintilla2 < Formula
   end
 
   option "without-plugin", "Skip building the Qt Designer plugin"
+  option "without-python", "Skip building the python bindings"
 
   depends_on :python => :recommended
```

Comment:
Then it's probably best to go back since the original formula was 100% backward compatible...